python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -123,6 +123,8 @@ def apply_filter_and_search(self, args):
         tsf_filter_term = args.get('tsf', None, type=str)
         # Base query:
         mines_query = Mine.query
+        # Adding a filter to remove deleted mines
+        mines_query = Mine.query
```

Comment:
It looks like you're immediately reassigning `mines_query` to the same value?

Reply:
Good catch I added this before I used apply_filters instead and forgot to remove it.